src/common/__tests__/data/errors.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * An instance of the `Error` class built in to JavasScript
+ */
+export const jsError = new Error('Plain JavaScript Error');
```

Comment:
I mentioned this is a reply to another comment,  but we do have an `error` already defined in our test data in `generics.ts`